plugins/inputs/cpufreq/cpufreq.go

```diff
@@ -0,0 +1,116 @@
+package cpufreq
```

Comment:
As this only works on linux, please make sure it only builds on linux.
```suggestion
// +build linux
package cpufreq
```

Reply:
@R4scal 
Moreover, you will need to add `cpufreq_notlinux.go` file with following content:
```
// +build !linux

package cpufreq

```

To build (but not include in binary) your plugin succesfully for non-Linux systems.

Look at systtat plugin: https://github.com/influxdata/telegraf/tree/f7d94430d29d15c42d8d186c99bfd065b9a21fa0/plugins/inputs/sysstat